lib/generators/react_on_rails/templates/client/package.json

```diff
@@ -0,0 +1,81 @@
+{
```

Comment:
We can also output text from a file, ie: README, at the end of the generation.

http://guides.rubyonrails.org/generators.html#readme